pyblnet/blnet.py

```diff
@@ -106,6 +106,51 @@ def _turn(self, digital_id, value, can_node=None):
         else:
             raise EnvironmentError('Can\'t set values with blnet web disabled')
 
+    def get_value(self,type='digital', ret='value', name=None, id=None, cached=None):
```

Comment:
I propose to move name and id to the front of the arguments, being the most important ones. After all `ret` and `cached` are optional while `name` or `id` aren't

Reply:
Done